dips/dip-10.md

```diff
@@ -51,75 +51,26 @@ Example: `alice@avasp`
 
-The Move contract that manages DiemID domains must emit an event every time DiemID domain is created, removed or updated. Those events are critical for applications to be able to efficiently index existing DiemID domains.
-An application can be built to listen for these events to construct a mapping of DiemID domains to VASP accounts for lookup of onchain addresses given a DiemID domain.
-While DiemID domains are published into VASP account resource, DiemID domain events are published under the TreasuryCompliance account. We consolidate events under single account to allow indexers to follow a single event stream.
-
-To store events, `DiemIdDomainManager` resource is published under the TreasuryCompliance account(address `0xB1E55ED`).
-
-```
-resource struct DiemIdDomainManager {
-    /// Events are emmitted 
-    diem_id_domain_events: Event::EventHandle<Self::DiemIdDomainEvent>,
-}
-
-struct DiemIdDomainEvent {
-    removed: bool,
-    domain: DiemIdDomain,
-    address: address,
-}
-```  
-
-* Fields definition:
-  * `removed` - `true`, if DiemIdDomain was removed, `false` if it was created or updated
-  * `domain` - exact copy of `DiemIdDomain` that was added/removed from `DiemIdDomains` resource of a VASP account
-  * `address` - address of an account where `DiemIdDomain` was added or removed
+Sender VASP may build an application to listen for domain events to construct a mapping of DiemID domains to VASP on-chain addresses to lookup onchain addresses given a DiemID domain.
 
 
 # ReferenceID Exchange
-Transactions with PaymentMetadata require a ReferenceID in order to submit a transaction to the chain. PaymentMetadata reveal nothing distinctly about either the sender or receiver and do not create a linkability between the sender or receiver across payments. Hence, the ReferenceID must be established off-chain and this protocol defines one such way to do so leveraging DiemID.
-
-A VASP intending to send a payment to another using DiemIDs constructs a ReferenceIDCommand and transmits it to the receiver. If the receiver knows the recipient DiemID and can potentially accept payments, it returns a success. Otherwise, it returns an error indicating the reason.
-
-In the case that the amount to be sent would exceed the limit of the travel rule, the sending party should follow this exchange with a PaymentCommand using the same reference_id and specify the sending and receiving subaddresses as all 0.
+Once the sender VASP gets the receiving VASP's address using on-chain domain lookup, the sender VASP initiates a reference ID exchange as defined in [DIP-183](https://github.com/diem/dip/blob/main/dips/dip-183.md)
+in order settle on a unique reference ID and submit the transaction on-chain.
 
 The format of the command is:
 
 ```
 {
    "_ObjectType": "CommandRequestObject",
-    "command_type": "ReferenceIDCommand",
+    "command_type": "SenderInitiatedReferenceID",
```

Comment:
If we are going to rename this command, I suggest new name "ExchangePaymentAddress".
The reason is, for this command, what really need is for sender to get receiver's account address, when sender got receiver's DiemID.
We originally thought reference id is the data need to be exchanged. But I think reference id is an id for referencing the payment that uses DiemID and related account addresses.
It is better to not assume this is a command initiate the reference id, because if we had another command that needs exchange information between VASPs, and the command happens before this command, then we may have trouble.

Reply:
Yeah, naming could definitely be improved!

Reply:
We can call it SenderInitiatedDiemIdExchange if we want to be specific to be diemID and not mention reference ID, since we're worried that reference ID exchange could happen in other commands

Reply:
I like this name `SenderInitiatedDiemIDExchange`